apps/admin-gui/src/app/facilities/pages/resource-detail-page/resource-detail-page.component.html

```diff
@@ -9,6 +9,20 @@ <h1 class="page-title">
     <span class="text-muted">
       &nbsp;#{{resource.id}}
     </span>
+    <br>
+    <span class="text-muted mt-1 entity-info">
+      {{'RESOURCE_DETAIL.VO' | translate}}:
+      <span class="resource-text">
+        {{resource.voId}} / {{resource.vo.name}}
+      </span>
+    </span>
+    <br>
+    <span class="text-muted mt-1 entity-info">
+      {{'RESOURCE_DETAIL.DESCRIPTION' | translate}}:
+      <span class="resource-text">
+        {{resource.description}}
+      </span>
+    </span>
```

Comment:
I would like another information about the facility, should look like info about vo
both facility and vo should be clickable and lead into the facility/vo details
also try to somehow make to one row - or just  try to use as less height as possible, because I think that this "sideinfo" takes too much space, if thats not looking good just do one information one row.